lucene/tools/forbiddenApis/solr.txt

```diff
@@ -55,3 +55,8 @@ com.google.common.base.Preconditions#checkNotNull(java.lang.Object,java.lang.Obj
 @defaultMessage Use methods in java.util.Comparator instead
 com.google.common.collect.Ordering
 
+@defaultMessage Use corresponding classes in package org.apache.solr.common.json instead
```

Comment:
I'd propose to just add `org.noggit.**`, this disallows whole package.

Reply:
sure. thanks . I didn't know it was possible